pkg/client/dtclient/client.go

```diff
@@ -295,6 +298,18 @@ func WithAutoServerVersion() func(client *DynatraceClient) {
 	}
 }
 
+// WithCachingDisabled lets disable the client builtin caching mechanism for
+// classic configs, schema constraints and settings objects
+func WithCachingDisabled(disabled bool) func(client *DynatraceClient) {
```

Comment:
As there's no way to re-enable the caches by setting the flag to `true`, and the name already states disabled I guess we could just remove the boolean?

Reply:
as discussed offline, let's keep it to avoid `if`s at the caller's side